src/utils/Types.ts

```diff
@@ -4,31 +4,39 @@ import type { Difficulty } from './Difficulty';
 
 export type RiddlePreview = {
 	id?: number;
+	//TODO: Make displays based on linkId instead of dbId
+	linkId: string;
```

Comment:
Link was moved here to become compulsory since we will probably use it instead of ID for details etc.